### ISSUE TITLE
Merge params for type without params

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#204]
Schema: 
```
type Query {
  test: Boolean
}

type Query {
  test(
    success: Boolean
  ): Boolean
}
```
Merge without errors, but total schema is
```
type Query {
  test: Boolean
}

```

Maybe this is not bug?... In this case required marge params for inner object (type Query inner type test). This is not "merge types", this "merge type's files". But for root types maybe usefull?
For this example probably: 
```
type Test {
  field: String
  field2: String
}

type Test {
  field2(
    param: String
  ): String
}
```
Merge params?

**Comment by kamilkisiela** [comment]
Should be fixed by https://github.com/ardatan/graphql-toolkit/pull/189

**Comment by kamilkisiela** [comment]
Fixed with `v1.7.0`
